app/controllers/notifications_controller.rb

```diff
@@ -49,15 +49,15 @@ def destroy
   end
 
   private
-    # Use callbacks to share common setup or constraints between actions.
-    def set_notification
-      @notification = Notification.find(params[:id])
```

Comment:
Changed this to `find_by_id` since `find` throws a 500 if record is not found. TODO: Redirect if `@notification` is not found.